pkg/descheduler/policyconfig.go

```diff
@@ -150,9 +150,15 @@ func validateDeschedulerConfiguration(in api.DeschedulerPolicy, registry pluginr
 				continue
 			}
 			if err := pluginUtilities.PluginArgValidator(pluginConfig.Args); err != nil {
-				errorsInProfiles = append(errorsInProfiles, fmt.Errorf("in profile %s: %s", profile.Name, err.Error()))
+				errorsInPolicy = append(errorsInPolicy, fmt.Errorf("in profile %s: %s", profile.Name, err.Error()))
 			}
 		}
 	}
-	return utilerrors.NewAggregate(errorsInProfiles)
+	if len(in.MetricsProviders) > 1 {
```

Comment:
What is the reason behind limiting the functionality? Do we want to enforce setting only a single source until the code base is ready for it?

Reply:
Yes. I will extend the test through https://github.com/kubernetes-sigs/descheduler/pull/1533.